### ISSUE TITLE
Just for funny,

### ISSUE THREAD

**Issue by keshiim** [dasdom/BreakOutToRefresh#1]
I`m sorry, there`s a awesome Proj, You are a genius.

**Comment by dasdom** [comment]
Thanks!

**Comment by danfinlay** [comment]
Yeah my issue is also that this is funny.  We should stop settling for spinners.  Nice work.

**Comment by dasdom** [comment]
Thank you!

**Comment by MHaroonBaig** [comment]
@dasdom An idea is just wandering around my mind, haha. What if you could integrate accelerometer and gyroscope to control the movement of the Bar in the small mini game that appears ? I've had written a very nice wrapper for CoreMotion here, https://github.com/MHaroonBaig/MotionKit
I'd love to send a pull request to your awesome project ;)

**Comment by dasdom** [comment]
@MHaroonBaig I had the same idea but didn't go further in that direction because BreakOutToRefresh is already using a lot CPU and memory.
Another problem is, at the moment the user of the app can continue to play when the loading finishes by keeping the finger on the display. In case of motion control this feature would be lost.
Maybe you can fork the project and implement it. I'll link to you implementation in the Readme. The people can chose what fits best their needs.

**Comment by MHaroonBaig** [comment]
Sure. I'll give it a try.

**Comment by Haoxiqiang** [comment]
This is so funny.Nice work.

**Comment by dasdom** [comment]
@Haoxiqiang Thank you!

**Comment by TellH** [comment]
my god！what a creative genius！

**Comment by dasdom** [comment]
@TellH Thank you!

**Comment by nghuiqin** [comment]
It's is so impressive!

**Comment by dasdom** [comment]
@baby03201 Thank you!

**Comment by amebahead** [comment]
I Like this Project that is inspired me

**Comment by dasdom** [comment]
Thank you!
